Tools/clinic/clinic.py

```diff
@@ -1367,6 +1361,14 @@ def parser_body(
                                             declarations=declarations)
 
 
+        # Copy includes from parameters to Clinic after parse_arg() has been
+        # called above.
+        for converter in converters:
+            for include in converter.includes:
+                assert include is not None
```

Comment:
I don't see the point of this assertion. If it's None, it will blow up below on `include.filename`.

Reply:
```suggestion
```